src/mesh/Router.cpp

```diff
@@ -378,6 +378,8 @@ bool perhapsDecode(meshtastic_MeshPacket *p)
         // parsing was successful
         p->which_payload_variant = meshtastic_MeshPacket_decoded_tag; // change type to decoded
         p->channel = chIndex;                                         // change to store the index instead of the hash
+        if (p->decoded.has_bitfield)
+            p->decoded.want_response |= p->decoded.bitfield & 1 << 1;
```

Comment:
Had to look up whether `<<` is actually before `&`, can we add brackets? And maybe a macro `BITFIELD_WANT_RESPONSE_SHIFT` and `BITFIELD_OK_TO_MQTT_SHIFT` (=0)?

Reply:
> Had to look up whether `<<` is actually before `&`, can we add brackets? And maybe a macro `BITFIELD_WANT_RESPONSE_SHIFT` and `BITFIELD_OK_TO_MQTT_SHIFT` (=0)?

Done.

Reply:
Don't want to be nitpicky here, but I think we should use them in `MQTT.cpp` as well, then it's immediately clear what it means.

Reply:
> Don't want to be nitpicky here, but I think we should use them in `MQTT.cpp` as well, then it's immediately clear what it means.

Lol, Came to that conclusion, too.